src/mono/wasm/debugger/DebuggerTestSuite/SteppingTests.cs

```diff
@@ -862,5 +862,54 @@ public async Task BreakpointOnHiddenLineOfMethodWithNoNextVisibleLineShouldNotPa
             Task t = await Task.WhenAny(pause_task, Task.Delay(2000));
             Assert.True(t != pause_task, "Debugger unexpectedly paused");
         }
+
+        [Fact]
+        public async Task SimpleStepTest()
```

Comment:
What is this test for?

Reply:
Trying to make sure that this issue is already fixed:
https://github.com/dotnet/runtime/issues/49141

Reply:
Rename it to something like `SimpleStep_RegressionTest_49141`.

Reply:
done.